eth2/_utils/bls/__init__.py

```diff
@@ -107,6 +94,9 @@ def validate(cls,
                  pubkey: BLSPubkey,
                  signature: BLSSignature,
                  domain: Domain) -> None:
+        if cls.backend != NoOpBackend:
+            validate_signature(signature)
+            validate_public_key(pubkey)
         if not cls.verify(message_hash, pubkey, signature, domain):
```

Comment:
nitpick: add a newline
```suggestion

        if not cls.verify(message_hash, pubkey, signature, domain):
```